nuspec/kre-clr-win-x64.nuspec

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package>
     <metadata>
-        <id>dotnet-clr-win-x64</id>
+        <id>kre-clr-win-x64</id>
```

Comment:
I think it's possible to generate this too. You can pass name value pairs to nuget.exe with the tokens you want to replace with. Might be worth seeing if that works here.